README.md

```diff
@@ -1,5 +1,6 @@
-# DigitalOcean Exporter [![Build Status](https://cloud.drone.io/api/badges/metalmatze/digitalocean_exporter/status.svg)](https://cloud.drone.io/metalmatze/digitalocean_exporter)
+# DigitalOcean Exporter [![Deploy to DO](https://mp-assets1.sfo2.digitaloceanspaces.com/deploy-to-do/do-btn-blue.svg)](https://cloud.digitalocean.com/apps/new?repo=https://github.com/metalmatze/digitalocean_exporter/tree/master)
```

Comment:
This button certainly is too big to go to the very top of the README.
I'd prefer this becoming another section under "Installation" in the README.

Reply:
Absolutely, fixed.

Reply:
![Screen Shot 2021-01-04 at 1 28 34 PM](https://user-images.githubusercontent.com/21261388/103571582-c60e1180-4e90-11eb-8b61-deb730ed13f1.png)

Reply:
![Screen Shot 2021-01-04 at 1 29 28 PM](https://user-images.githubusercontent.com/21261388/103571669-e6d66700-4e90-11eb-8d7f-f9106eb47f79.png)